util/topgen.py

```diff
@@ -270,7 +276,7 @@ def generate_plic(top, out_path):
             fpv_tpl = Template(fin.read())
             try:
                 out = fpv_tpl.render(src=src, target=target, prio=prio)
-            except:
+            except:  # noqa : E722
```

Comment:
is this comment a possible error message?

Reply:
That is to waive flake8. Mako template needs unspecified `except` clause.

Reply:
oh that is a lint waiver then, got it. can you maybe add a comment on the line above saying that this is a lint waiver and it is needed?

Reply:
I've put a number of this comment to topgen already. Is it okay to put at the very beginning of the script file for the error message not put the explanatory comment everywhere?

Reply:
sure, as long as there is a comment about this somewhere in this file ;).